src/main/java/io/appium/java_client/touch/offset/ElementOption.java

```diff
@@ -37,6 +39,19 @@ public static ElementOption element(WebElement element) {
     }
 
     /**
+     * It defines x and y offset from the upper left corner of an element.
```

Comment:
of an element (if provided) of the whole screen (if the element is not provided)